src/core/send.c

```diff
@@ -1120,6 +1119,17 @@ QuicSendFlush(
         return TRUE;
     }
 
+    //
+    // Connection CID changes on idle state after an amount of time
+    //
+    uint64_t TimeNow = CxPlatTimeUs64();
```

Comment:
Move this up, and use it's value for like 1109, where it calls `QuicMtuDiscoveryCheckSearchCompleteTimeout`.